friends/src/components/NavBar.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  UncontrolledDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem
+} from "reactstrap";
```

Comment:
Great job pushing yourself to complete the stretch goals, and tyring something new like `reactstrap` 👍